front/src/app/todo-list/todo-list.component.ts

```diff
@@ -48,21 +49,53 @@ export class TodoListComponent implements OnInit, OnDestroy {
 
   taskListInputValue: string = '';
 
+  resize$ = fromEvent(window, 'resize');
+
+  sidebarMode: MatDrawerMode = 'side';
+
+  isSidebarOpen: boolean = true;
+
   private destroy$ = new Subject<void>();
 
   constructor(private taskListManager: TaskListManagerService, private apiService: ApiService, private changeDetectorRef: ChangeDetectorRef, private router: Router) {}
 
   ngOnInit(): void {
+    this.isSidebarOpen = document.documentElement.clientWidth >= 750

```

Comment:
`document.documentElement.clientWidth >= 750` is used in multiple places, avoid this duplication